src/classes/Handsontable.php

```diff
@@ -0,0 +1,830 @@
+
+    /**
+     * ######################################
+     * Handsontable JS library attributes
+     * ######################################
+     */
+
+    /**
+     *
+     * @see https://docs.handsontable.com/latest/Options.html#cell
+     */
+    protected $cell = null;
+
+    /**
+     *
+     * @see https://docs.handsontable.com/latest/Options.html#cells
+     */
+    protected $cells = null;
+
+    /**
+     *
+     * @see https://docs.handsontable.com/latest/Options.html#startRows
+     */
+    protected $startRows;
+
+    /**
+     *
+     * @see https://docs.handsontable.com/latest/Options.html#startCols
+     */
+    protected $startCols;
+
+    /**
+     *
+     * @see https://docs.handsontable.com/latest/Options.html#data
+     */
+    protected $data = null;
+
+    /**
+     *
+     * @see https://docs.handsontable.com/latest/Options.html#columns
+     */
+    protected $columns = null;
+
+    /**
+     *
+     * @see https://docs.handsontable.com/latest/Options.html#width
+     */
+    protected $width = null;
+
+    /**
+     *
+     * @see https://docs.handsontable.com/latest/Options.html#height
+     */
+    protected $height = null;
+
+    /**
+     *
+     * @see https://docs.handsontable.com/latest/Options.html#rowHeaders
+     */
+    protected $rowHeaders = true;
+
+    /**
+     *
+     * @see https://docs.handsontable.com/latest/Options.html#colHeaders
+     */
+    protected $colHeaders = true;
+
+    /**
+     *
+     * @see https://docs.handsontable.com/latest/Options.html#dataSchema
+     */
+    protected $dataSchema = null;
+
+    /**
+     *
+     * @see https://docs.handsontable.com/latest/Options.html#contextMenu
+     */
+    protected $contextMenu = ['row_above', 'row_below', 'remove_row', 'undo', 'redo', 'copy', 'cut'];
+
+    /**
+     *
+     * @see https://docs.handsontable.com/latest/Options.html#autoWrapRow
+     */
+    protected $autoWrapRow = true;
+
+    /**
+     *
+     * @see https://docs.handsontable.com/latest/Options.html#minSpareRows
+     */
+    protected $minSpareRows = 1;
+
+    /**
+     *
+     * @see https://docs.handsontable.com/latest/Options.html#maxCols
+     */
+    protected $maxCols;
+
+    /**
+     *
+     * @see https://docs.handsontable.com/latest/Options.html#rowHeights
+     */
+    protected $rowHeights;
+
+    /**
+     *
+     * @see https://docs.handsontable.com/latest/Options.html#colWidths
+     */
+    protected $colWidths;
+
+    /**
+     *
+     * @see https://docs.handsontable.com/latest/Options.html#sortIndicator
+     */
+    protected $sortIndicator = false;
+
+    /**
+     *
+     * @see https://docs.handsontable.com/latest/Options.html#manualColumnResize
+     */
+    protected $manualColumnResize = false;
+
+    /**
+     *
+     * @see https://docs.handsontable.com/latest/Options.html#manualRowResize
+     */
+    protected $manualRowResize = false;
+
+    public function __construct($container_name = null, $cellConfig = null)
```

Comment:
In this constructor, a few things are done. This is not a simple constructor.
You should add the constructor documentation to explain what is done.

Reply:
Done